### ISSUE TITLE
pom.xml:32-34: Need to fix other build errors in order to...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/s3auth#371]
The puzzle `317-34dbec36` from #317 has to be resolved:

https://github.com/yegor256/s3auth/blob/66b7324bf580062cc5509f3f591907ed2ed066a3/pom.xml#L32-L34

The puzzle was created by George Aristy on 14-Apr-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3RP1J1CH) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#371](https://github.com/yegor256/s3auth/issues/371) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#371](https://github.com/yegor256/s3auth/issues/371) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C3RP1J1CH/1132119018); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting because I delayed before starting this

**Comment by 0crat** [comment]
> @0crat waiting because I delayed before starting this ([here](https://github.com/yegor256/s3auth/issues/371#issuecomment-385472327))

@llorllale The impediment for [#371](https://github.com/yegor256/s3auth/issues/371) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@0crat waiting on teamed/qulice#891

**Comment by 0crat** [comment]
> @0crat waiting on teamed/qulice#891 ([here](https://github.com/yegor256/s3auth/issues/371#issuecomment-385499242))

@llorllale Job [#371](https://github.com/yegor256/s3auth/issues/371) is already on hold

**Comment by llorllale** [comment]
@0crat refuse (the classpath issues raised by this upgrade are too complex I guess)

**Comment by 0crat** [comment]
> @0crat refuse (the classpath issues raised by this upgrade are too complex I guess) ([here](https://github.com/yegor256/s3auth/issues/371#issuecomment-393559505))

@llorllale The user @llorllale[/z](https://www.0crat.com/u/llorllale) resigned from [#371](https://github.com/yegor256/s3auth/issues/371), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
Job `gh:yegor256/s3auth#371` is not assigned, can't get performer
